pkg/controller/pagerdutyintegration/clusterdeployment_created.go

```diff
@@ -93,7 +94,7 @@ func (r *ReconcilePagerDutyIntegration) handleCreate(pdclient pd.Client, pdi *pa
 	}
 
 	pdData := &pd.Data{
-		ClusterID:          cd.Spec.ClusterName,
+		ClusterID:          clusterNS[len(clusterNS)-1],
```

Comment:
you brought this up but yeah I think it would be cleaner to have the clusterID as separate var here so we can re-use it if needed.

Reply:
yes makes sense.